public/views/estimation/estimation_experiment.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+  <title>VCL: Estimation Experiment</title>
+
```

Comment:
Add in `<div id="cm_conversion" style = "visibility: hidden; width: 100cm; height: 100cm;"></div>` for conversion div, then inside script below can do:
```
 const PIXELS_PER_CM = document.getElementById("cm_conversion").getBoundingClientRect().height/100;
```
Then you can pass that conversion factor into the params